### ISSUE TITLE
Accessibility: units

### ISSUE THREAD

**Issue by paavovaananen-cgi** [fmidev/weather-app#363]
Unit changes should be accessible to screen reader. At the moment screen reader reads the "default" units. This can be implemented after #347 is merged.
